dashboard/app/controllers/maker_controller.rb

```diff
@@ -1,6 +1,4 @@
 class MakerController < ApplicationController
-  authorize_resource class: :maker_discount, except: [:home, :setup, :login_code, :display_code, :confirm_login]
```

Comment:
Can you explain a bit why we can remove routes like `login_code` when they're not using the MakerDiscount class? I don't know what they do and the comments aren't super clear to me.